Makefile

```diff
@@ -14,7 +14,7 @@ help:
 	@echo
 	@echo 'Usage:'
 	@echo '    make build           Compile the project.'
-	@echo '    make get-deps        runs glide install, mostly used for ci.'
+	@echo '    make get-deps        runs dep ensure, mostly used for ci.'
```

Comment:
Small change, this could be confusing since the package is using go mod rather than the tool dep.

Reply:
fixed